device_backends/LogicalNameMapping/src/LNMBitRangeAccessPlugin.cc

```diff
@@ -206,6 +193,7 @@ namespace ChimeraTK::LNMBackend {
     ReferenceCountedUniqueLock _lock;
     VersionNumber _temporaryVersion;
     bool _writeable{false};
+    FixedPointConverter fixedPointConverter;
```

Comment:
since this is a struct, the others are wrong, but then again I do not remember why its a struct. I will leave it for now and it should be reconsidered for a next round of formal code cleaning